src/main/ruby/core/truffle/cext.rb

```diff
@@ -1397,12 +1397,12 @@ def rb_jump_tag(pos)
   end
 
   def rb_yield(value)
```

Comment:
Could you check if these changes allow this spec to be untagged?
https://github.com/graalvm/truffleruby/blob/9f3216ef068878a91afadba6980de8007be5821f/spec/tags/optional/capi/kernel_tags.txt#L2

Reply:
I'll check, I've removed the rb_block_proc fix from this PR for now though. It's getting complicated to fix properly.